LibraryApi/Repository/UserRepository.cs

```diff
@@ -0,0 +1,40 @@
+using LibraryApi.Entity;
+using Microsoft.EntityFrameworkCore;
+
+namespace LibraryApi.Repository;
+
+public class UserRepository : IUserRepository
+{
+    private readonly LibraryContext context;
```

Comment:
Por conveção de código, propriedades de somente leitura (readonly) recebem um prefixo `_`
```suggestion
    private readonly LibraryContext _context;
```

Reply:
certo resolvido